johnsonandjohnson/Bodiless-JS
# fix(migration-tool): postbuild should not trim ssi elements from generated html

## Changes
* fixed the issue regarding trimming ssi elements from generated html during post build phase
* make html cleanup only during flattening. disable html cleanup on postbuild phase
* remediated old unit tests
* updated unit tests

## Test Instructions
1. Activate ssi on psh
2. Add migrate:postbuild to static site build
3. Deploy the static site to psh
Expected: ssi elements are present in the generated html

## Related Issues
